FeaturevisorDemo/FeaturevisorDemo.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,322 @@
+// !$*UTF8*$!
```

Comment:
@polok: is this a must-have to keep in the repo? or can we `.gitignore` it?

I haven't run any serious Xcode project before, and don't know myself if this is a common practice in the Swift world. I will trust your judgement here.

Reply:
Well, it's autogenerated by Xcode. It is responsible for maintaining references to all of the linked files and their groupings, linked frameworks, and most importantly, the project’s build settings. Without it, we won't be able to run it.